client/gutenberg/extensions/simple-payments/editor.js

```diff
@@ -13,13 +12,16 @@ import edit from './edit';
 import save from './save';
 import { DEFAULT_CURRENCY } from 'lib/simple-payments/constants';
 import { __, _x } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import registerJetpackBlock from 'gutenberg/extensions/presets/jetpack/utils/register-jetpack-block';
 
 /**
  * Styles
  */
 import './editor.scss';
 
-registerBlockType( 'jetpack/simple-payments', {
+export const name = 'simple-payments';
+
+export const settings = {
```

Comment:
why do we export `name` and `settings`, do we use them somewhere else?

Reply:
No, not at the moment. But @simison  suggested this pattern in case other presets want to use them without all of the Jetpack logic. I thought it was a fine suggestion!

Reply:
It's core Gutenberg pattern to export them regardless of whether we use them. Webpack should be smart enough to treeshake and exclude whatever we don't use.

Reply:
Noting that we could see if relying on `index.json` and `index-beta.json` as a source of truth for Calypso imports would work. These exports are useful for that.

Much like this:

https://github.com/WordPress/gutenberg/blob/0006987541d7ac7c373ebc4b3e97870d5d6a4e63/packages/block-library/src/index.js#L97-L103